bin/review-rot

```diff
@@ -62,22 +62,24 @@ def main(state_, value, duration, ssl_verify=None):
                 """
                 res = format_user_repo_name(data)
                 """
-                get pull requests for specified git service, username
-                and repository name
+                get pull/merge/change requests for specified git service
                 """
+                repo_name =  data if res.get('repo_name') is None else res.get('repo_name')
                 results.extend(
                     git_service.request_reviews(
-                        user_name=res['user_name'],
-                        repo_name=res['repo_name'],
+                        user_name=res.get('user_name',None),
+                        repo_name=repo_name,
                         state_=state_,
                         value=value,
                         duration=duration,
-                        token=item.get('token'),
+                        token=item.get('token',None),
                         host=item.get('host'),
                         ssl_verify=ssl_verify,
                     )
                 )
 
+
```

Comment:
@vineetpalan Can you fix lint errors? May be use flake8? (That's what I had been using in past)